.github/scripts/build_ffmpeg_plugin.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+#!/bin/bash
+# filepath: /home/labrat/lgrab/SVT-JPEG-XS/ffmpeg-plugin/build_ffmpeg_svtjpegxs.sh
+
+set -e
+
+# Usage: ./build_ffmpeg_svtjpegxs.sh <ffmpeg-version: 6.1|7.0>
+JPEGXS_REPO=$(pwd)
+FFMPEG_VERSION=${1:-6.1}
+INSTALL_FMPEG=${2:-"n"}
+
+if [[ "$FFMPEG_VERSION" != "6.1" && "$FFMPEG_VERSION" != "7.0" ]]; then
+    echo "Usage: $0  <ffmpeg-version> <install-ffmpeg>"
+    echo "ffmpeg-version: 6.1|7.0"
+    echo "install-ffmpeg: y|n (default: n)"
+    echo "Example: $0 6.1 y"
+    exit 1
+fi
+echo "=== 0. Create installation directory and export env variable ==="
+export INSTALL_DIR="$PWD/install-dir"
+mkdir -p "$INSTALL_DIR"
+
+echo "=== 1. Compile and install svt-jpegxs ==="
+cd "$JPEGXS_REPO/Build/linux"
+./build.sh install --prefix "$INSTALL_DIR"
+
+echo "=== 2. Export installation location ==="
+export LD_LIBRARY_PATH="$INSTALL_DIR/lib:${LD_LIBRARY_PATH}"
+export PKG_CONFIG_PATH="$INSTALL_DIR/lib/pkgconfig:${PKG_CONFIG_PATH}"
+
+echo "=== 3. Download/Compile FFmpeg ==="
+cd "$PWD"
+git clone https://git.ffmpeg.org/ffmpeg.git ffmpeg-$FFMPEG_VERSION
+cd ffmpeg-$FFMPEG_VERSION
```

Comment:
SC2086 (info): Double quote to prevent globbing and word splitting.
```suggestion
git clone https://git.ffmpeg.org/ffmpeg.git "ffmpeg-${FFMPEG_VERSION}"
cd "ffmpeg-${FFMPEG_VERSION}"
```